server/routes/api/groups/groups.test.ts

```diff
@@ -258,6 +258,39 @@ describe("#groups.list", () => {
         .includes(anotherUser.id)
     ).toBe(true);
   });
+
+  it("should allow to find a group by its name", async () => {
+    const user = await buildUser();
+    const group = await buildGroup({
+      teamId: user.teamId,
+    });
+    const anotherGroup = await buildGroup({
+      teamId: user.teamId,
+    });
+
+    const unfilteredRes = await server.post("/api/groups.list", {
```

Comment:
It doesn't feel neccessary to include this portion of the test, as it's testing just the general list functionality